### PR TITLE
CounterButtonApp task implementation

### PR DESCRIPTION
**Pull request by Svetlana-Gubina**
Create vue app for CounterButtonApp, add button click handler.
Implement VueCalculator (with computed property).
Implement MarkedList task (with computed property).

### CODE REVIEW

**Comment by jsru** [comment]
Добавляю преподавателя (@ShGKme) для код-ревью.

**Comment by jsru** [comment]
Решение было обновлено, посмотрим что скажет @ShGKme

**Comment by jsru** [comment]
Решение было обновлено, посмотрим что скажет @ShGKme

File: 01-basics/02-VueCalculator/index.html [hunk 1]
```diff
@@ -31,23 +31,23 @@
...
 
         <div class="col" style="display: flex; flex-direction: column; font-family: emoji">
-          <label><input type="radio" name="operator" value="sum" /> ➕</label>
-          <label><input type="radio" name="operator" value="subtract" /> ➖</label>
-          <label><input type="radio" name="operator" value="multiply" /> ✖</label>
-          <label><input type="radio" name="operator" value="divide" /> ➗</label>
+          <label><input type="radio" name="operator" value="sum" @change="handleOperatorChange" /> ➕</label>
+          <label><input type="radio" name="operator" value="subtract" @change="handleOperatorChange" /> ➖</label>
+          <label><input type="radio" name="operator" value="multiply" @change="handleOperatorChange"/> ✖</label>
+          <label><input type="radio" name="operator" value="divide" @change="handleOperatorChange" /> ➗</label>
```
Comment (ShGKme) [inline 01-basics/02-VueCalculator/index.html]: All possible parts of the visible state of an application must be defined as the application state and be managed by the application. 

Here:
1. The state of radio buttons is not bound to the application state. Changing `operation` doesn't lead to changing of a visually checked radio button. `checked` state of every button must be set with `v-bind`.
2. `v-bind` + `v-on` (`@change`) can be shortened with two-way binding directive `v-model`.

**Review by ShGKme — CHANGES_REQUESTED** [review]

File: 01-basics/02-VueCalculator/index.html (hunk 1 above)
Comment (Svetlana-Gubina) [reply 01-basics/02-VueCalculator/index.html]: Fixed with v-model

**Comment by jsru** [comment]
Добавляю преподавателя (@ShGKme) для код-ревью.

**Review by ShGKme — APPROVED** [review]
Approved

**Pull request merged**